packages/runtime/src/next/tests/client.get-resources.test.ts

```diff
@@ -0,0 +1,245 @@
+import { Makeswift } from '../client'
+import { http, HttpResponse } from 'msw'
+
+import { ReactRuntime } from '../../runtimes/react'
+import { MakeswiftSiteVersion } from '../../api/site-version'
+
+import { server } from '../../mocks/server'
+
+const TEST_API_KEY = 'xxx'
+const apiOrigin = 'https://api.fakeswift.com'
+const runtime = new ReactRuntime()
+
+function createTestClient() {
+  return new Makeswift(TEST_API_KEY, { runtime, apiOrigin })
+}
+
+let consoleErrorSpy: jest.SpyInstance
+
+beforeEach(() => {
+  consoleErrorSpy = jest.spyOn(console, 'error').mockImplementation(() => {})
+})
+
+afterEach(() => {
+  consoleErrorSpy.mockRestore()
+})
+
+describe('getSwatch', () => {
```

Comment:
Error handling tests for `getSwatch`, `getTypography`, `getGlobalElement` et al.